### PR TITLE
SPCTR-2938 Fix: Table of content - Heading list HTML structure not proper.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-2938

### Screenshots

- https://share.getcloudapp.com/nOuLgyyd

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Added to the card.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: blocks-config/table-of-content/class-uagb-table-of-content.php [hunk 1]
```diff
@@ -344,9 +344,9 @@ public function table_of_contents_render_list(
 				}
 
 				if ( $li_added ) {
-					$toc .= sprintf( '<a href="#%s">%s</a></li>', esc_attr( $id ), esc_html( $title ) );
+					$toc .= sprintf( '<li class="uagb-toc__list"><a href="#%s">%s</a>', esc_attr( $id ), esc_html( $title ) );
```
Comment (adi3890) [inline blocks-config/table-of-content/class-uagb-table-of-content.php:347]: do we need if ... else here

File: readme.txt [hunk 2]
```diff
@@ -166,6 +166,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 == Changelog ==
 
+= x.x.x =
+* Fix: Table of content - Heading list HTML structure not proper.
```
[the remark this answers is not in the document]
Comment (AbhijitNage123) [reply readme.txt]: Resolved

**Review by adi3890 — APPROVED** [review]

File: readme.txt [hunk 3]
```diff
@@ -168,6 +168,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 = x.x.x =
 * Fix: Buttons - Resolved an issue in which the button size would not work as expected in the editor after adding padding.
+* Fix: Table of content - Fixed improper HTML markup/structure for Heading list.
```
Comment (bsf-zanev) [inline readme.txt:171]:
```suggestion
* Fix: Table Of Contents - Fixed the improper HTML markup/structure for the heading list.
```

**Review by bsf-zanev — APPROVED** [review]

**Review by aapatre — APPROVED** [review]
Preapproved, please add the above suggestion by @bsf-zanev 

cc: @AbhijitNage123

File: readme.txt (hunk 3 above)
Comment (AbhijitNage123) [reply readme.txt]: Done

**Pull request merged**